public/js/components/subscription.jsx

```diff
@@ -16,7 +16,7 @@ export default class Subscription extends Component {
     var productData = products.get(this.props.seller_product.public_id);
 
     return (
-      <div className="subscription">
```

Comment:
Hmm, yeah maybe. I needed a hook on the li - but yes this could potentially be re-used without the list. I'll add that to my list to iterate on this and re-instate this class and target the li element directly for the bits I needed.